### PR TITLE
1.24.1

### PR DESCRIPTION
**Pull request by math-GH**

### CODE REVIEW

File: versions.php [hunk 1]
```diff
@@ -49,7 +49,8 @@
 	'1.22.1' => '1.23.1',
 	'1.23.0' => '1.23.1',
 	'1.23.1' => '1.24.0',
```
Comment (Alkarex) [inline versions.php:51]:
```suggestion
	'1.23.1' => '1.24.1',
```

**Comment by Alkarex** [comment]
Ping @marienfressinaud

**Comment by beerisgood** [comment]
I know this was already discussed in the last update, but I would very much appreciate it if you would immediately release the internal update process as soon as the data is uploaded. So far, another two days have passed since the upload and we can't start the update.

I understand that a release is released delayed in order not to roll out possible bugs to all users, but then waiting for a second waiting time via manual release is, in my opinion, unnecessary work and waiting time.

**Comment by math-GH** [comment]
> I know this was already discussed in the last update, but I would very much appreciate it if you would immediately release the internal update process as soon as the data is uploaded. So far, another two days have passed since the upload and we can't start the update.
> 
> I understand that a release is released delayed in order not to roll out possible bugs to all users, but then waiting for a second waiting time via manual release is, in my opinion, unnecessary work and waiting time.

Quick fix for you:

As far as I can oversee it: It is a very simple php script. You can put it on your own server and change the update URL via config.php (see: https://freshrss.github.io/FreshRSS/en/admins/17_configs_not_ui.html ).

**Comment by marienfressinaud** [comment]
Normally, this version would have been available as soon as this PR has been merged. Unfortunately I was not at home these last days and I will not be able to update the server before tomorrow.

We could discuss to automatically deploy this repository to the server, as we did for the website, but we should discuss potential security issues first.

**Comment by beerisgood** [comment]
> Quick fix for you:
> 
> As far as I can oversee it: It is a very simple php script. You can put it on your own server and change the update URL via config.php (see: https://freshrss.github.io/FreshRSS/en/admins/17_configs_not_ui.html ).

Which URL is needed?
Currently I have:
`'auto_update_url' => 'https://update.freshrss.org',`

**Comment by Alkarex** [comment]
@beerisgood Why don't you deploy with git, which is also supported by the Web update? The ZIP update is the poorest of all install options we have (slowest, least robust, least possibility to change branch, least possibility to test features). The ZIP install is mostly meant for people with only FTP access to their shared hosting.

**Comment by beerisgood** [comment]
> @beerisgood Why don't you deploy with git, which is also supported by the Web update? The ZIP update is the poorest of all install options we have (slowest, least robust, least efficient in terms of resources, least possibility to change branch, least possibility to test features). The ZIP install is mostly meant for people with only FTP access to their shared hosting.

I like to use the UI so I don't have to execute manual commands via SSH.

According to [instructions](https://freshrss.github.io/FreshRSS/en/admins/04_Updating.html) the update via git is not easier than the zip variant - which I followed with this [guide](https://lab.uberspace.de/guide_freshrss/).

**Comment by Alkarex** [comment]
> I like to use the UI

The UI works also (best?) with git

**Comment by beerisgood** [comment]
> The UI works also (best?) with git

How?

**Comment by Alkarex** [comment]
> How?

I am not sure about your question, but if installed with git, the UI will also propose updates and will be able to apply them, and all that as soon as the release is done, without having to wait for the manual ZIP method

**Comment by marienfressinaud** [comment]
@beerisgood The update should be available now

**Comment by beerisgood** [comment]
Update was successful 🍺

**Pull request merged**